scripts/verify_build.sh

```diff
@@ -34,6 +37,7 @@ curl "https://packages.chef.io/manifests/dev/automate/latest.json" > results/dev
 curl "https://packages.chef.io/manifests/current/automate/latest.json" > results/current.json
 curl "https://packages.chef.io/manifests/acceptance/automate/latest.json" > results/acceptance.json
 
+: <<'BUILD_ALL'
```

Comment:
I think technically we have some feature for this that you could use if you wanted: https://github.com/chef/automate/blob/master/scripts/changed_components.rb#L30

Reply:
Dope, there's so much spaghetti here that I have no idea what I'm doing

Reply:
> there's so much spaghetti

For sure.  I feel like there are two solutions. (1) fix it (2) lean into it and just rename the directory _TrattoriaDiShell_ or something.